db/migrations/manual/20221222-partially-adj-synthea-rel1-runout-update.sql

```diff
@@ -0,0 +1,14 @@
+-- Updating the sandbox runout files for partially adjudicated TEST99x accounts
+-- to allow runout testing.
+
+BEGIN;
+UPDATE cclf_files
+    set type=1,
+        updated_at=now(),
+        timestamp='2022-09-29',
```

Comment:
Not sure if timestamp year should be updated, or if it needs to match the year in the name set, question for BCDA

Reply:
i suggest we update one variable at a time; to see if this works as expected

Reply:
i wonder about the performance_year
will this stop working on 1/1/2023 ?